src/pages/data-visualization/chart-types/index.mdx

```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
     href="/data-visualization/basic-charts#pie"
     >
 
-![Pie](images/donut.svg)
+![](images/pie.svg)
 
 </OverviewCard>
 </Column>
+
 <Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
  <OverviewCard
+    title="Stacked bar (horizontal)"
+    href="/data-visualization/basic-charts/#stacked-bar"
+    >
+
+![](images/stacked-bar-horizontal.svg)
+
+</OverviewCard>
+</Column>
+<Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
+ <OverviewCard
+    title="Stacked area"
+    href="/data-visualization/basic-charts/#stacked-area-chart"
     title="Stacked bar (vertical)"
-    href="/data-visualization/basic-charts/#stacked-bar-(discrete)"
+    href="/data-visualization/basic-charts/#bar-(vertical)"
```

Comment:
this doesn't look like it'll actually scroll down to the stacked bar demo, it'll go to the **Bar (vertical)** heading